nodes/cma/nodes-cma-autoscaling-custom-trigger-auth.adoc

```diff
@@ -15,50 +15,47 @@ Alternatively, to share credentials between objects in multiple namespaces, you
 
 Trigger authentications and cluster trigger authentication use the same configuration. However, a cluster trigger authentication requires an additional `kind` parameter in the authentication reference of the scaled object.
 
-.Example secret for Basic authentication
+.Example trigger authentication that uses a bound service account token
 [source,yaml]
 ----
-apiVersion: v1
-kind: Secret
+kind: TriggerAuthentication
+apiVersion: keda.sh/v1alpha1
 metadata:
-  name: my-basic-secret
-  namespace: default
-data:
-  username: "dXNlcm5hbWU=" <1>
-  password: "cGFzc3dvcmQ="
+  name: secret-triggerauthentication
+  namespace: my-namespace <1>
+spec:
+  boundServiceAccountToken: <2>
+    - parameter: token
+      serviceAccountName: my-service-account <3>
```

Comment:
We are referencing `oc create serviceaccount thanos` in the first step, so we should be consistent and use it here too.

Reply:
Outdated